R/render.R

```diff
@@ -86,8 +86,18 @@ quarto_render <- function(input = NULL,
   if (as_job && rstudioapi::isAvailable()) {
     message("Rendering project as background job (use as_job = FALSE to override)")
     script <- tempfile(fileext = ".R")
+    render_args <- as.list(sys.call()[-1L])
+    render_args <- mapply(
+      function(arg, arg_name) paste0(
+        arg_name,
+        "="[nchar(arg_name) > 0L],
+        deparse1(eval(arg, envir = parent.frame(n = 3L)))
+      ),
+      render_args,
+      names(render_args)
+    )
```

Comment:
I think we need to handle this a bit differently for  complex wrapping calls that would use `...` for example. In this case, it will not work. **rlang** maybe be useful for this. 🤔

Reply:
That's right, I didn't test with a wrapper function that uses dots, I only tested simple named args. Feel free to refactor and expand on this PR as you see fit!